Pipfile

```diff
@@ -14,4 +15,4 @@ jinja2 = "*"
 redis = "*"
 
 [requires]
-python_version = "3.6"
+python_version = "3.7"
```

Comment:
Will need to change the python version in travis as it will still be pointing at python 3.6